Sources/ImageCache+Rx.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ImageCache+Rx.swift
+//  RxKingfisher-iOS
+//
+//  Created by Amit Kumar Swami on 4/3/19.
+//  Copyright © 2019 RxSwift Community. All rights reserved.
+//
+
+import RxSwift
+import Kingfisher
+
+extension Reactive where Base: ImageCache {
+    public func retrieveImageInDiskCache(
+        forKey key: String,
+        options: KingfisherOptionsInfo? = nil) -> Single<Image?> {
+        return Single<Image?>.create { [base] single in
+            base.retrieveImageInDiskCache(forKey: key, options: options) { result in
+                switch result {
+                case let .failure(error):
```

Comment:
Could you change this and the success case to have the `let` on the argument? I personally use a `case let` only if there's more than one argument.

```suggestion
                case .failure(let error):
```